src/System.IO.FileSystem.AccessControl/tests/DirectoryObjectSecurityTests.cs

```diff
@@ -0,0 +1,232 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System.DirectoryServices;
+using System.Security.Principal;
+using Xunit;
+
+namespace System.Security.AccessControl
+{
+    public class DirectoryObjectSecurityTests
+    {
+        [Fact]
+        public void ObjectInitialization_DefaultConstructor_Success()
+        {
+            var activeDirectorySecurity = new ActiveDirectorySecurity();
+            Assert.NotNull(activeDirectorySecurity);
+        }
+
+        [Fact]
+        public void ObjectInitialization_CommonSecurityDescriptor_Success()
+        {
+            
+            CommonSecurityDescriptor descriptor = new CommonSecurityDescriptor(false, false, string.Empty);
+            var customObjectSecurity = new CustomDirectoryObjectSecurity(descriptor);
+            Assert.NotNull(customObjectSecurity);
```

Comment:
Same `new` comment as earlier.  Won't comment on it further, but it applies to all other cases.